src/main/resources/config.properties

```diff
@@ -13,6 +13,7 @@ builder.matlab.automatictestoption.display.name = Automatic
 builder.matlab.customcommandoption.display.name = Custom
 Releaseinfo.matlab.version.not.found.error = Error finding MATLAB release for given MATLAB root. Verify MATLAB root path.
 matlab.not.found.error = Unable to launch MATLAB from the specified location. Verify the path to MATLAB root folder.
+matlab.not.found.error.for.node = Unable to launch MATLAB registered with the name %s on the node %s. Verify global tool configuration for the specified node.
```

Comment:
How about shortening the first sentence this way (assuming the second sentence clarifies that it is a tool)?

Unable to launch MATLAB %s on the node %s.

Reply:
Applied the suggested edits.